www/src/design-system/elements/Button.mdx

```diff
@@ -0,0 +1,106 @@
+---
+name: Button
+---
+
+import { Box, Button } from "@artsy/palette"
+
+# Button
+
+## primaryBlack
+
+<Playground title="primaryBlack">
+  <>
```

Comment:
Haven't figured out how to embed adjacent content in and hide first level of nesting (the fragment), since the contents of playground are what end up being rendered in the preview, not playground + contents. Pretty trivial issue in the end since most content is just a single component... could be confusing for folks tho.